server/controller/src/main/java/ai/starwhale/mlops/schedule/impl/docker/RunExecutorDockerImpl.java

```diff
@@ -131,10 +150,14 @@ public void onComplete() {
                     createContainerCmd.withCmd(containerCommand.getCmd());
                 }
                 CreateContainerResponse createContainerResponse = createContainerCmd.exec();
+                dockerClient.connectToNetworkCmd()
+                        .withNetworkId(networkName)
+                        .withContainerId(createContainerResponse.getId())
+                        .exec();
```

Comment:
How does the controller connect to the container through this network?

Reply:
`host` network which is the default behavor is kept to be supported after commit [keep support for host network](https://github.com/star-whale/starwhale/pull/3041/commits/99c76a63a69545fc89c24bc7a8675e62469ee514) .
If user choose to use network other than `host` he/she should guarentee that controller is in the same network as did by https://github.com/tianweidut/starwhale/blob/a6dd437b92d91bcf6a4d375b82643361feb702c3/client/starwhale/cli/server/compose.tmpl